cluster/cluster.go

```diff
@@ -58,10 +59,19 @@ func (c *Cluster) DeployClusterPlanes() error {
 	err = services.RunControlPlane(c.ControlPlaneHosts,
 		c.EtcdHosts,
 		c.Services,
-		c.SystemImages[ServiceSidekickImage])
+		c.SystemImages[ServiceSidekickImage],
+		c.Authorization.Mode)
 	if err != nil {
 		return fmt.Errorf("[controlPlane] Failed to bring up Control Plane: %v", err)
 	}
+	if err = authz.ApplyJobExecuterServiceAccount(c.LocalKubeConfigPath); err != nil {
```

Comment:
can we extract RBAC config call into a separate method? Would be easier to read.